docs/source/concepts/ledger-model/ledger-exceptions.rst

```diff
@@ -122,6 +122,84 @@ And the third:
 
 As you can see, in each of these continuities, no contract was consumed twice.
 
+Transaction Normalization
++++++++++++++++++++++++++
+
+The same "before-after" relation can be represented in more than one way using
+rollback nodes. For example, the following three transactions have the same
+"before-after" relation among their ledger actions (A and B):
```

Comment:
Elsewhere in the ledger model, we use upper-case letters `A` and `B` to represent parties. Actions are typically called `act` with indices.
```suggestion
"before-after" relation among their ledger actions (`act1` and `act2`):
```